index.html

```diff
@@ -311,72 +330,163 @@ <h2>The <dfn>PerformanceObserver</dfn> interface</h2>
+            <a data-cite="HTML/webappapis.html#concept-relevant-global">relevant
+            global object</a> contains a <a>registered performance observer</a>
+            <var>obs</var> whose <a>observer</a> is the <a data-cite=
+            "WHATWG-DOM#context-object">context object</a>:
+              <ol>
+                <li>If <var>obs</var>'s <a>options list</a> contains a
+                <a>PerformanceObserverInit</a> item <var>currentOptions</var>
+                whose <a>type</a> is equal to <var>options</var>'s <a>type</a>,
+                replace <var>currentOptions</var> with <var>options</var> in
+                <var>obs</var>'s <a>options list</a>.</li>
+                <li>Otherwise, append a new
+                <a>registered performance observer</a> object to the <a>list of
+                registered performance observer objects</a> of
+                <a data-cite="HTML/webappapis.html#concept-relevant-global">
+                relevant global object</a>, with <a>observer</a> set to the
+                <a data-cite="WHATWG-DOM#context-object">context object</a> and
+                <a>options list</a> set to a list containing <var>options</var>
+                as its only item.</li>
+              </ol>
+            </li>
+            <li>If <var>options</var>'s <a>buffered</a> flag is set:
+              <ol>
+                <li>Let <var>entries</var> be the <a>PerformanceEntryList</a>
+                object returned by the <a href="#filter-buffer-by-name-and-type">
+                </a> algorithm with <var>buffer</var> set to <a>relevant
+                performance entry buffer</a>, <var>name</var> set to
+                <code>null</code> and <var>type</var> set to
+                <var>options</var>'s <a>type</a>.
+                </li>
+                <li>Append <var>entries</var> to the <a data-cite=
+                "WHATWG-DOM#context-object">context object</a>'s <a>observer
+                buffer</a>.
+                </li>
+              </ol>
+            </li>
+          </ol>
+          </li>
         </ol>
+        <p class="note">A <a>PerformanceObserver</a> object needs to always call
+        <a>observe()</a> with <var>options</var>'s
+        <a data-link-for=PerformanceObserverInit>entryTypes</a> set OR always
+        call <a>observe()</a> with <var>options</var>'s
+        <a data-link-for=PerformanceObserverInit>type</a> set. This is meant to
+        avoid confusion with how calls would stack. When using
+        <a data-link-for=PerformanceObserverInit>entryTypes</a>, no other
+        parameters in <a>PerformanceObserverInit</a> can be used. In addition,
+        multiple <a>observe()</a> calls will override for backwards
+        compatibility and because a single call should suffice in this case. On
+        the other hand, when using
+        <a data-link-for=PerformanceObserverInit>type</a>, calls will stack
```

Comment:
You also want to add a note that calling `observe()` with entry types and then `observe()` with type (and vice versa) will throw

Reply:
Also, when calling observe with the same type multiple times, it will override, right?